CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
  - Run linting only on affected and changed examples
  - Add SequenceEffect
  - Fixed bug with travelTime in RotateEffect
+ - Upgrade tiled and flutter_svg dependencies
```

Comment:
Is develop outdated, check how the CHANGELOG is on master: https://github.com/flame-engine/flame/blob/master/CHANGELOG.md#0220

Reply:
well I'm just adding my line to the changelog, check my diff. If there is a desync between develop and master that is not concern for this PR I think

Reply:
oh I see my develop was out of date, rebased again. check now. I thought github would compare with current develop

Reply:
> I thought github would compare with current develop

Lol, I thought this too